### PR TITLE
Release: 2015.10.8

### PR DESCRIPTION
**Pull request by krak3n**
Initial release of `Flask-HAL`
- Version bump to `2015.10.8`
- Added a Contributors File
- Updated Change Log
- Included Change Log & Contributors in documentation index.

### CODE REVIEW

**Comment by krak3n** [comment]
@peeklondon / @radeklos I think this covers it :smile: 

Can you check i've not missed anything?

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+sys.path.append(os.path.join(root, 'flask_hal'))
+
+# First Party Libs
+import flask_hal  # noqa
+
+# Project details
+project = u'Flask-HAL'
+copyright = u'{0}, SOON_ London Ltd'.format(datetime.datetime.utcnow().year)
+version = '2015.10.8'
```
Comment (radeklos) [inline docs/conf.py]: This should be probably loaded from the version file.

Comment (krak3n) [inline docs/conf.py]: @radeklos sorted.

**Comment by radeklos** [comment]
:+1:

**Pull request merged**